files/galaxy/config/user_preferences_extra_conf.yml

```diff
@@ -111,3 +111,11 @@ preferences:
               label: Password
               type:  password
               required: False
+
+    c3s_account:
+        description: Your CDS API Key (Copernicus Climate Change Service API Key)
+        inputs:
+            - name: c3s_cds_apikey
+              label: C3S CDS API Key
+              type: text
```

Comment:
password?

Reply:
No there is just a key (it is a long string). I guess it is like a password (it is unique per user).

Reply:
but this string is secret it should not be visible by your colleague dropping by your screen with a photographic memory? :)

Reply:
Yes. It should not be shared. I did not think about it! Is it sufficient to replace `type: text` by `type: password`? Do I need to update the tool too?

Reply:
No, that is suficient. type=password just means here that galaxy will display **** and such in the UI